examples/xml-functions/xml_functions.bal

```diff
@@ -31,23 +29,12 @@ public function main() {
     x = content.elements();
     io:println(x);
 
-    // Retrieves an XML element by its name.
-    x = content.select("name");
-    io:println(x);
-
     // Sets the children elements of an XML element.
     book.setChildren(content);
     io:println(book);
 
-    // Retrieves a particular child of an XML element by its name.
-    x = book.selectDescendants("name");
-    io:println(x);
-
-    // Removes any text items from an XML sequence that are all whitespaces.
+    // Strips the insignificant parts of the an xml value.
+    // Comment items, processing instruction items are considered insignificant.
```

Comment:
```suggestion
    // Commented items and processing instruction items are considered insignificant.
```

Reply:
Comment items are a thing in XML (<!-- this is a comment item -->)

Reply:
+1